src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -158,6 +158,26 @@ public function getFieldChoices(FormView $view): iterable
         yield from $this->createFieldChoicesList($view->vars['choices'], $view->vars['choice_translation_domain']);
     }
 
+    /**
+     * @internal
```

Comment:
Maybe we can put a comment that this function has been copied from `twig_test_empty`, to help track changes.

Reply:
Or maybe we use `CoreExtension::testEmpty()` if it exists?

Reply:
[`CoreExtension::testEmpty()` is internal](https://github.com/twigphp/Twig/blob/3.x/src/Extension/CoreExtension.php#L1322), its signature could change, or it could be moved or renamed.

Reply:
Given that we intend to migrate Twig into a Symfony component, I think it's fine to access `@internal` functions.

Reply:
Even in Symfony components, it is not fine to use `@internal` APIs of a *different* component, as components relies on semver ranges in their inter-component requirements.